components/Input/Input.stories.tsx

```diff
@@ -117,40 +118,55 @@ ReadOnly.args = { readOnly: true, defaultValue: 'value' };
 
 export const Ghost: ComponentStory<typeof InputForStory> = (args) => (
   <Flex direction="column" gap={2}>
-    <Label>
-      Small
+    <Box>
+      <Label htmlFor="small">
+        Small
+      </Label>
       <InputForStory size="small" {...args} />
-    </Label>
-    <Label>
-      Default
-      <InputForStory {...args} />
-    </Label>
-    <Label>
-      Large
-      <InputForStory size="large" {...args} />
-    </Label>
-    <Label>
-      Invalid
-      <InputForStory state="invalid" {...args} />
-    </Label>
-    <Label>
-      Disabled
-      <InputForStory disabled {...args} />
-    </Label>
-    <Label>
-      ReadOnly
-      <InputForStory readOnly {...args} />
-    </Label>
-    <Label>
-      Adornments
+    </Box>
+    <Box>
+      <Label htmlFor="default">
+        Default
+      </Label>
+      <InputForStory id="default" {...args} />
```

Comment:
On the docs part of the storybook, all these stories are shown on the same page, which causes a conflict on the ids, triggering multiple tooltips at once. It doesn't affect the usability of the lib, but I think we should avoid this behavior as our storybook is public.

https://user-images.githubusercontent.com/38889364/143313849-3187fac4-bc90-4c72-a9ec-4fd3f7976b7d.mov


Another example on Tooltips:

https://user-images.githubusercontent.com/38889364/143313406-7fe9b0fc-faf8-4570-bd97-25f6d2f5802f.mov

Reply:
```suggestion
      <InputForStory id="ghost-default" {...args} />
```

Reply:
@paulocfjunior This issue happens because `IdProvider` is wrapping each story on its own instead of wrapping the whole docs once. The consequence for tooltips is that `id`s are duplicated between each story.

I create a separate PR to handle this issue

Reply:
Fixed for TextField, Label and Input stories: https://github.com/traefik/faency/pull/197/commits/7b40b78294080a5ceb528a7c54ad574860259fbe

:warning: I didn't notice that each story title held an `id` which may conflict with the ids I was setting...
![image](https://user-images.githubusercontent.com/3367393/143486030-2e62c24a-5c55-4db6-accf-4dc93db0fcb0.png)

@matthieuh @paulocfjunior Any idea how we could ensure we don't fall into this trap again?